JavaScript/JavaScript.sublime-syntax

```diff
@@ -1363,8 +1370,24 @@ contexts:
     - match: (?={{class_element_name}})
       push: class-element
 
+  static-block:
+    - match: static{{identifier_break}}
+      scope: storage.modifier.js
+      set:
+        - match: \{
+          scope: punctuation.section.block.begin.js
+          set:
+            - meta_scope: meta.block.js
```

Comment:
Especially as this syntax is heavily used as base for inheriting, I'd vote to avoid anonymous contexts whenever possible. It also helps debugging now that context names are printed to the scope popup.

Reply:
I've added `static-block-body` for the most deeply-nested context.